bindings/java/src/README.md

```diff
@@ -22,4 +22,16 @@ To skip integration tests, execute `ctest -E integration` from `${BUILD_DIR}/bin
 To run _only_ integration tests, run `ctest -R integration` from `${BUILD_DIR}/bindings/java`.
 
 There are lots of other useful `ctest` commands, which we don't need to get into here. For more information,
-see the [https://cmake.org/cmake/help/v3.19/manual/ctest.1.html](ctest documentation).
\ No newline at end of file
+see the [https://cmake.org/cmake/help/v3.19/manual/ctest.1.html](ctest documentation).
+
+### Multi-Client tests
+Multi-Client tests are integration tests that can only be executed when multiple clusters are running. To write a multi-client
+test, do the following:
+
+1. Tag all tests that require multiple clients with `@Tag("MultiClient")`
+2. Ensure that your tests have the `MultiClientHelper` extension present, and Registered as an extension
+3. Ensure that your test class is in the the JAVA_INTEGRATION_TESTS list in `test.cmake`
+
+It is important to note that it requires significant time to start and stop 3 separate clusters; if the underying test takes a long time to run,
```

Comment:
There is 2 second sleep before we configure the new database which is probably more than we should have.

https://github.com/apple/foundationdb/blob/master/tests/loopback_cluster/run_cluster.sh#L44

Reply:
in my book, 4 seconds qualifies as "significant time" :)

the default timeout on JUnit 5 is about 60 seconds, which is ample time to run the vast majority of tests, even spawning clusters. However, I added this note specifically because I was encountering test timeouts that I had to account for in the test itself. For the most part, it shouldn't be a problem, but it needs to be accounted for when writing tests.